certified-connectors/DocuSignDemo/script.csx

```diff
@@ -2374,8 +2383,10 @@ public class Script : ScriptBase
 
       var crmOrgUrl = query.Get("crmOrgUrl") ?? null;
       var recordId = query.Get("recordId") ?? null;
-      var crmType = "CRMToken";
-      string[] filters = { crmType, recordId, crmOrgUrl};
+      var crmType = query.Get("crmType").ToString().Equals("Dynamics365") ? "CRMToken" : "SFToken";
```

Comment:
Does there need to be a check for null on the results of query.Get?

Reply:
crmType is a dropdown with only 2 options. It is either Dynamics365 or salesforce. Need not check for null here.